AnnoyancesFilter/sections/widgets.txt

```diff
@@ -301,6 +301,7 @@ full-count.jp#?#section > div.c-heading:has(> p.c-heading__en:contains(RECOMMEND
 .ru/proxy_*.php?p___=*&part_url=*.php?callback=*&token=*&iso=
 .net/proxy_*.php?p___=*&part_url=*.php?callback=*&token=*&iso=
 !
+||whatya.solairo-api.com/aoscs/chat.js
```

Comment:
Apparently the whole domain is for chat so `||whatya.solairo-api.com^$third-party` will be okay.

Reply:
Not sure about that. As I see, that bot provides useful info.

Reply:
Well, most of online consultant widgets have some utility. What is the criteria to block or not to block?

Reply:
If they really annoying. We do not block literally all widgets with this purpose.
We will change the policy for annoyances a bit later.

Reply:
After a couple of days, we'll split the Annoyances filter apart and the problem will be less serious.